components/infra/docker-compose.yml

```
@@ -71,34 +113,68 @@ services:
+      start_period: 15s
     env_file:
       - .env
+    # Inline env mapping required: Valkey/Redis image uses REDIS_USER and
+    # REDIS_PASSWORD for runtime auth configuration.
     environment:
       - REDIS_USER=${REDIS_USER}
       - REDIS_PASSWORD=${REDIS_PASSWORD}
+    # NOTE: Password is passed via command args (visible in process list).
+    # This is acceptable for local development only. In production, use a config file
+    # mounted as a volume or environment-based secret injection.
     command: [ "redis-server", "--requirepass", "${REDIS_PASSWORD}",  "--port", "${REDIS_PORT}" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)